_delphi_utils_python/data_proc/geomap/zip_pop_filling.txt

```diff
@@ -0,0 +1,25 @@
+zip,pop
```

Comment:
are both .txt and .csv needed?

Reply:
no sorry, I though I have removed the txt file. Now it is done.